### PR TITLE
terraform improvements

### PR DESCRIPTION
**Pull request by eschultink**
### Features
 - expose variable to control JAR version at top level
 - bulk file test script for GCP

### Change implications

 - dependencies added/changed? **no**

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204191088895030
  - https://app.asana.com/0/0/1204210840874631
  - https://app.asana.com/0/0/1204227472646368

### CODE REVIEW

File: infra/modular-examples/aws-msft-365/variables.tf [hunk 1]
```diff
@@ -76,6 +82,12 @@ variable "psoxy_base_dir" {
   }
 }
 
+variable "bundle_version" {
```
Comment (jlorper) [inline infra/modular-examples/aws-msft-365/variables.tf:85]: couldn't this just be read from a file in the root?

Comment (eschultink) [reply infra/modular-examples/aws-msft-365/variables.tf:85]: yeah, probably

File: infra/modules/psoxy-package/main.tf [hunk 2]
```diff
@@ -39,5 +31,9 @@ output "path_to_deployment_jar" {
 }
 
 output "filename" {
-  value = local.filename
+  value = data.external.deployment_package.result.filename
+}
+
+output "version" {
```
Comment (jlorper) [inline infra/modules/psoxy-package/main.tf:37]: I think `version` can be removed, as far as I can tell is not used and value comes embedded in filename.

**Review by jlorper — APPROVED** [review]
OK, I think `version` is redundant as output and could be removed, the less outputs to propagate the better. But no blocker obviously.

**Comment by jlorper** [comment]
Good improvement by the way!

File: infra/modules/psoxy-package/main.tf (hunk 2 above)
Comment (eschultink) [reply infra/modules/psoxy-package/main.tf:37]: i think it's good to have the structured data there if we ever need it,

**Pull request merged**